compiler/passes/errorHandling.cpp

```diff
@@ -285,6 +285,9 @@ bool ErrorHandlingVisitor::enterCallExpr(CallExpr* node) {
 
       VarSymbol* errorVar    = NULL;
       BlockStmt* errorPolicy = new BlockStmt();
+      Expr*      insert      = node->getStmtExpr();
+      if (insert == NULL)
+        insert = node;
```

Comment:
I don't have a problem with this going in as a bug fix, but I don't really understand why parentExpr can be NULL while node->insertBefore will still work, for a CallExpr.